src/electron/electron/url.cljs

```diff
@@ -64,6 +64,12 @@
       (= "auth-callback" url-host)
       (send-to-renderer win "loginCallback" (.get (.-searchParams parsed-url) "code"))
 
+      (= "quick-capture" url-host)
+      (let [[url title description] (get-URL-decoded-params parsed-url ["u" "t" "d"])]
```

Comment:
Could we use full names for the url params? It's more readable for maintainers and users

Reply:
will address this